cli/src/cli.rs

```diff
@@ -120,11 +117,16 @@ pub enum Commands {
     },
     /// Show the current status of the p2p network.
     Network,
-    /// Serve the p2p network indefinitely.
+    /// Become a server node indefinitely, serving all message propagation and Git requests.
+    ///
+    /// You can not perform any other operations while running this command;
+    /// you have to run another shell to perform client-side, synchronous operations.
     Serve,
-    /// Fetch the data broadcasted over the network and update it to the repository,
-    /// the governance, and the consensus.
-    Fetch,
+    /// Update the node state by fetching data from the p2p network,
+    /// verifying incoming data, and applying to the repository and consensus & governance status.
+    Update,
```

Comment:
Why rename?

Reply:
because I realized that the command is supposed perform more task than just 'fetching' data. (verification, application)

Reply:
But doesn't git fetch do the same way? "Fetching", verifying then applying.

Reply:
We should rename them `update` as well, I guess.